src/data.js

```diff
@@ -0,0 +1,115 @@
+    imageUrl: `pictures/hospitals/CHU_Mustapha_Pacha.jpg`
+  },
+  {
+    id: uuid(),
+    title:`CHU Beni Messous`,
+    location: `36°46'49.2"N 2°58'53.0"E`,
+    // imageUrl: `https://upload.wikimedia.org/wikipedia/commons/9/99/CHU_Beni_Messous_%D8%A7%D9%84%D9%85%D8%B3%D8%AA%D8%B4%D9%81%D9%89_-_panoramio_%283%29.jpg`
+    imageUrl: `pictures/hospitals/CHU_Beni_Messous.jpg`
+  },
+  {
+    id: uuid(),
+    title:`CHU Bab El Oued`,
+    // imageUrl: `https://upload.wikimedia.org/wikipedia/commons/5/57/Hopital_Mohamed_Lamine_Debaghine_%C3%A0_Alger.jpg`,
+    imageUrl: `pictures/hospitals/CHU_Bab_El_Oued.jpg`,
+    location: `36°47'36.3"N 3°03'02.5"E`
+  },
+  {
+    id: uuid(),
+    title:`EHS Salim Zemirli`,
+    // imageUrl: `https://scontent-mad1-1.xx.fbcdn.net/v/t1.0-9/39162712_519421175174545_7685812511261589504_o.jpg?_nc_cat=100&_nc_ohc=aRczq-r9z0sAQleCHo7rAwhM66s-QmzP9-7bvIyYcVmeMpZSR-erEATOg&_nc_ht=scontent-mad1-1.xx&oh=20d35f20f4826a616469c64a1ad5ba8e&oe=5E7FEC16`,
+    imageUrl: `pictures/hospitals/EHS_Salim_Zemirli.jpg`,
+    location: `36°47'36.3"N 3°03'02.5"E`
+  },
+  {
+    id: uuid(),
+    title:`EHS Dr Madouche (CNMS)`,
+    // imageUrl: `https://lh3.googleusercontent.com/p/AF1QipM0OdgQrJIJturhn4vytJMSJdLJB1Y-ZHwdM3wW=s1600-h380`,
+    imageUrl: `pictures/hospitals/EHS_CNMS.jpg`,
+    location: `36°46'05.0"N 3°00'01.2"E`
+  },
+  {
+    id: uuid(),
+    title:`CHU Douera`,
+    // imageUrl: `https://scontent-mad1-1.xx.fbcdn.net/v/t1.0-9/s960x960/62783938_2348426542112483_8161708876366872576_o.jpg?_nc_cat=106&_nc_ohc=3gjzgRlPyBgAQlm-7Xq0iIBZAvDghwXWTJnkFL4RQve5xGJ67gQql1SYw&_nc_ht=scontent-mad1-1.xx&oh=2d20ba6ea5db4be29adfc01c090da815&oe=5E4F4A49`,
+    imageUrl: `pictures/hospitals/CHU_Douera.jpg`,
+    location: `36°39'58.7"N 2°56'34.1"E`
+  },
+  {
+    id: uuid(),
+    title:`EHS Ait Idir`,
+    // imageUrl: `http://www.alger-city.com/pics/ville/sante/hopital-ali-ait-idir/hopital-neurochirurgie-ali-ait-idir-casbah-alger.jpg`,
+    imageUrl: `pictures/hospitals/EHS_Ali_Ait_Idir.jpg`,
+    location: `36°47'13.6"N 3°03'29.7"E`
+  },
+  {
+    id: uuid(),
+    title:`EPH El Biar (Birtraria)`,
+    // imageUrl: `https://scontent-mad1-1.xx.fbcdn.net/v/t31.0-8/p960x960/12240912_138925849801153_2814322703766458871_o.jpg?_nc_cat=111&_nc_ohc=CpYuTgpA38sAQlSXDEIuaF4Ffj8XcAyuf9wXkTHPbxPOoen5Iif2yUF9w&_nc_ht=scontent-mad1-1.xx&oh=392050154e07d8b5e7a6f1776a643e10&oe=5E4E82DB`,
+    imageUrl: `pictures/hospitals/EPH_Birtraria.jpg`,
+    location: `36°46'21.6"N 3°02'24.3"E`
+  },
+  {
+    id: uuid(),
+    title:`EPH Kouba`,
+    // imageUrl: `https://upload.wikimedia.org/wikipedia/commons/f/ff/Photo_hopital_kouba_26052016.jpg`,
+    imageUrl: `pictures/hospitals/EPH_Kouba.jpg`,
+    location: `36°43'32.7"N 3°05'14.0"E`
+  },
+  {
+    id: uuid(),
+    title:`EPH Bologhine`,
+    imageUrl: `https://cdnfr2.img.sputniknews.com/images//104213/97/1042139753.jpg`,
+    imageUrl: `pictures/hospitals/EPH_Bologhine.jpg`,
```

Comment:
Why there is two `imageUrl` attributes ?

Reply:
There is only one imageUrl attributes, but in two ways: 
1.st way that is **Commented**, where we have the link of the picture.
2.nd  way where the picture are saved and we indicate the relative path.

We'll use the second way, but i kept the first thinking that we might need them until we finally deploy our last version. 
The mistake here is that i forgot to comment all the imageUrl that has link redirecting to the picture. that's why we see two imageUrl, but it's solved now in 1633967